origin.spec

```diff
@@ -367,35 +375,36 @@ fi
 
 
 %preun master
-%systemd_preun %{basename:%{name}-master.service}
+%systemd_preun %{name}-master.service
 
 %postun master
 %systemd_postun
 
 %files node
-%defattr(-,root,root,-)
 %{_unitdir}/%{name}-node.service
 %config(noreplace) %{_sysconfdir}/sysconfig/%{name}-node
 %config(noreplace) %{_sysconfdir}/origin/node
+%ghost %config(noreplace) %{_sysconfdir}/origin/.config_managed
 
 %post node
-%systemd_post %{basename:%{name}-node.service}
+%systemd_post %{name}-node.service
 
 %preun node
-%systemd_preun %{basename:%{name}-node.service}
+%systemd_preun %{name}-node.service
 
 %postun node
 %systemd_postun
 
 %files sdn-ovs
-%defattr(-,root,root,-)
+%dir %{_unitdir}/docker.service.d/
+%dir %{_unitdir}/%{name}-node.service.d/
```

Comment:
If we own both of these and someone else places files in them do we delete them on uninstall?